generators/app/templates/src/app/reducer.ts

```diff
@@ -0,0 +1,48 @@
+import { Nullable } from '~utils/types';
+
+export interface User {
+  id: number;
+}
+
+export interface GlobalState {
+  user: Nullable<User>;
+}
+
+export const INITIAL_STATE = {
+  user: null
+};
+
+enum ActionTypes {
+  SET_USER = 'SET_USER',
+  RESET_USER = 'RESET_USER'
+}
+
+export interface SetUser {
+  type: ActionTypes.SET_USER;
+  payload: User;
+}
+
+export interface ResetUser {
```

Comment:
Do you need to export individual action types?

Reply:
probably not... at least not right now. I'll remove it